modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_entry_usages/page.jsp

```diff
@@ -23,7 +23,9 @@ String className = GetterUtil.getString(request.getAttribute("liferay-asset:asse
 AssetEntryUsagesDisplayContext assetEntryUsagesDisplayContext = new AssetEntryUsagesDisplayContext(renderRequest, renderResponse, className, classPK);
 %>
 
-<div class="container-fluid container-fluid-max-xl container-form-lg">
+<clay:container
+	className="c-pb-md-5"
```

Comment:
Is this a type? Shouldn't it be `c-pt-mb-5` like in https://github.com/eduardoallegrini/liferay-portal/pull/70/files#diff-8dfe0f21bf041bdc6a670f09cacb2b8cR37, or is that one wrong?

It seems easier to write and remember `container-form-lg` than this, and certainly harder to typo...